### PR TITLE
Panels: text updates

### PR DESCRIPTION
**Pull request by breville**
This tidies up the code and styling related to showing text in panels, and adds a small slide-in movement to the text box to unify it with the Lab2 instructions styling.

https://github.com/user-attachments/assets/24cd9cc9-99a9-4338-a943-2eec286ff877

### CODE REVIEW

File: apps/src/panels/PanelsView.tsx [hunk 1]
```diff
@@ -208,6 +210,7 @@ const PanelsView: React.FunctionComponent<PanelsProps> = ({
       >
         {showButton && (
           <Button
+            key={'button-' + currentPanelIndex}
```
Comment (thomasoniii) [inline apps/src/panels/PanelsView.tsx:213]: both this line and line 159 really should be written as template strings, as preferred modern syntax.

e.g., `button-${currentPanelIndex}`.

There's also an eslint rule that would enforce it (and I'm pretty sure also automatically clean it up), if we wanted to enable it.

**Review by thomasoniii — APPROVED** [review]
minor request to use template strings, but otherwise lgtm 🚀

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/panels/PanelsView.tsx (hunk 1 above)
Comment (breville) [reply apps/src/panels/PanelsView.tsx:213]: Ah good point.  Might do this more broadly in a follow-up!

Comment (breville) [reply apps/src/panels/PanelsView.tsx:213]: Fixed in https://github.com/code-dot-org/code-dot-org/pull/62208.

**Pull request merged**